### PR TITLE
Fix Chinese translation for playlist downloader and add Chinese translation for the installer

### PR DESCRIPTION
**Pull request by Aruvelut-123**
that's it.

### CODE REVIEW

File: YoutubePlaylistDownloader/Languages/中文.xaml [hunk 1]
```diff
@@ -3,163 +3,180 @@
...
+    <s:String x:Key="DownloadSpeed">下载速度：</s:String>
+    <s:String x:Key="Changelog">更新日志</s:String>
+    <s:String x:Key="DongChunrui">Dong Cunrui (董存瑞)</s:String>
+    <s:String x:Key="CreditsAndContributors">感谢这些帮助翻译此软件的人：
 Dong Cunrui (董存瑞)
+Aruvelut-123
 RichyZ99
 Jhonatan Lima (Heremitus)
 İskender ALPSAKARYA, Alp Eren GENÇOĞLU (Türkçe)
-Luis Duenas (Español)
```
Comment (shaked6540) [inline YoutubePlaylistDownloader/Languages/中文.xaml:105]: Why did you remove this credit?

**Comment by shaked6540** [comment]
Was the Chinese translation wrong? I see that you changed most of it

File: YoutubePlaylistDownloader/Languages/中文.xaml (hunk 1 above)
Comment (Aruvelut-123) [reply YoutubePlaylistDownloader/Languages/中文.xaml:105]: IDK，it's accident i think，let me put it back

**Comment by Aruvelut-123** [comment]
Because most of it are bot translate

File: YoutubePlaylistDownloader/Languages/中文.xaml (hunk 1 above)
Comment (Aruvelut-123) [reply YoutubePlaylistDownloader/Languages/中文.xaml]: OK i add it back already

Comment (Aruvelut-123) [reply YoutubePlaylistDownloader/Languages/中文.xaml]: IDK why i translating this will accidently delete some of these

**Comment by Aruvelut-123** [comment]
Fixed accident remove qwq
